### ISSUE TITLE
MPI support

### ISSUE THREAD

**Issue by mingtaiha** [radical-cybertools/radical.synapse#16]
Will Synapse have any profiling capabilities for application running MPI in the near future?

**Comment by andre-merzky** [comment]
No, and given the plethora of MPI monitoring and profiling solutions, it is probably not adviseable to roll our own.  We could consider integrating one (or several) of those tools, or to consume external MPI profiles for our emulation purposes.
